index.html

```diff
@@ -454,19 +205,22 @@ <h2 class="ski">Skills</h2>
               touch.
             </article>
```

Comment:
- [ ] [REQUIRED] The `embuza` class text is aligned to the center while it is aligned left on the template provided. Kindly implement this so as to make sure we are following all the requirements. 
#### Your Implementation
![2](https://user-images.githubusercontent.com/58692590/155815785-b869ea4a-f164-461c-a42f-9472de05133e.png)

#### Provided Template
![22](https://user-images.githubusercontent.com/58692590/155815808-f1aa6852-1a11-4ad1-95f9-bd741ee5064f.png)